hasher-matcher-actioner/hmalib/models.py

```diff
@@ -47,12 +47,12 @@ def remove_content_key_prefix(key: str) -> str:
         return key[len(DynamoDBItem.CONTENT_KEY_PREFIX) :]
 
 
```

Comment:
I renamed *SNS* / *sns* to *AWS* / *aws* because we're using the same functionality in both SNS and SQS contexts.